remote-config.jsonc

```
@@ -123,6 +123,30 @@
         },
         {
           "message": "Did you know you can set environment variables with the `.ddev/.env` file?"
+        },
+        {
+          "message": "You can contribute your own tips to these daily tips with a PR on https://github.com/ddev/remote-config"
+        },
+        {
+          "message": "DDEV's goal is to make developers happy. We hope you're happy today!"
+        },
+        {
+          "message": "If you're on macOS you may want to try the excellent new Docker provider OrbStack, https://orbstack.dev"
```

These sentences that end with commas and links read strangely to me. I’d change the punctuation:

```suggestion
          "message": "If you're on macOS you may want to try the excellent new Docker provider OrbStack: https://orbstack.dev"
```